files/emr-apps.repo

```diff
@@ -0,0 +1,7 @@
+[emr-applications]
+name = EMR Applications Repository
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-emr
```

Comment:
So this is some key that's needed to fetch EMR?

Reply:
yes.